src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs

```diff
@@ -185,11 +187,9 @@ public override bool Execute ()
 			}
 
 			int apiLevel;
-			if (int.TryParse (AndroidApiLevel, out apiLevel)) {
-				if (apiLevel < 26)
-					Log.LogCodedWarning ("XA0113", $"Google Play requires that new applications must use a TargetFrameworkVersion of v8.0 (API level 26) or above. You are currently targeting {TargetFrameworkVersion} (API level {AndroidApiLevel}).");
+			if (AndroidApplication && int.TryParse (AndroidApiLevel, out apiLevel)) {
 				if (apiLevel < 26)
-					Log.LogCodedWarning ("XA0114", $"Google Play requires that application updates must use a TargetFrameworkVersion of v8.0 (API level 26) or above. You are currently targeting {TargetFrameworkVersion} (API level {AndroidApiLevel}).");
+					Log.LogCodedWarning ("XA0113", $"Google Play requires that new applications and updates must use a TargetFrameworkVersion of v8.0 (API level 26) or above. You are currently targeting {TargetFrameworkVersion} (API level {AndroidApiLevel}).");
```

Comment:
We will need to update the documentation to reflect the change in wording, and probably delete the documentation for `XA0114` if it is no longer required.

Reply:
Relevant documentation files which would need to be updated are:

  * https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/messages/README.md
  * https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/messages/xa0113.md

Reply:
@dellis1972 If you delete the doc, what happens if users are using the older tooling and can't get info on the error?
Do you have people who are good at writing doc for stuff like this? It goes a bit beyond what I intended

Reply:
> If you delete the doc, what happens if users are using the older tooling and can't get info on the error?

I don't understand the question.

Additionally, at present most of the docs in this repo are "advisory," to the extent that they're not on `docs.microsoft.com`.  (We intend to fix this, but it hasn't happened yet.)

> It goes a bit beyond what I intended.

I believe Dean was only requesting that the existing docs that we have -- which I linked to [above](https://github.com/xamarin/xamarin-android/pull/2754#discussion_r259022091) -- be updated to use the same message you altered in this PR, for consistency.  That is all.

Reply:
Regarding deleting doc... It's still possible for users to get XA0114 as long as people are on older versions of the build task. If I delete the doc, people looking for information on that bug will immediately be unsuccessful once this PR merges.

I'll fix the message in XA0113...